tests/test_schema_errors/test_strings.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+
+from openapi_tester import DocumentationError, SchemaTester
+
+example_schema_string = {"type": "string", "minLength": 3, "maxLength": 5}
+example_string = "str"
+
+tester = SchemaTester()
+
+
+# region: string
```

Comment:
whats the point of these comments?

Reply:
They tell your IDE to create collapsible sections of code. Makes it easier to separate sections.

![image](https://user-images.githubusercontent.com/25310870/106355458-4b51de00-62f8-11eb-8bda-0319dad8db85.png)

Reply:
i see, interesting. Thanks!